test/algorithm.test.js

```diff
@@ -0,0 +1,355 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable func-names */
+import { expect } from 'chai';
+import { mochaAsync, openDrawer, clearInput } from './utils';
+import {
+  ALGORITHMS_MENU_ITEM_ID,
+  ALGORITHM_TABLE_ID,
+  ALGORITHM_NAME_CLASS,
+  ALGORITHM_DESCRIPTION_CLASS,
+  ALGORITHM_AUTHOR_CLASS,
+  ALGORITHM_LANGUAGE_CLASS,
+  ALGORITHM_DELETE_BUTTON_CLASS,
+  buildAlgorithmRowClass,
+  ALGORITHM_ADD_BUTTON_ID,
+  ADD_ALGORITHM_NAME_ID,
+  ADD_ALGORITHM_DESCRIPTION_ID,
+  ADD_ALGORITHM_FILE_LOCATION_ID,
+  ADD_ALGORITHM_SAVE_BUTTON_ID,
+  ADD_ALGORITHM_BACK_BUTTON_ID,
+  ADD_ALGORITHM_FROM_FILE_OPTION_ID,
+  ADD_ALGORITHM_FROM_EDITOR_OPTION_ID,
+  ALGORITHM_EDIT_BUTTON_CLASS,
+  EDIT_ALGORITHM_NAME_ID,
+  EDIT_ALGORITHM_DESCRIPTION_ID,
+  EDIT_ALGORITHM_SAVE_BUTTON_ID,
+  EDIT_ALGORITHM_BACK_BUTTON_ID,
+} from '../src/config/selectors';
+import { createApplication, closeApplication } from './application';
+import { DEFAULT_GLOBAL_TIMEOUT } from './constants';
+import {
+  GRAASP_ALGORITHMS,
+  MISSING_FILE_ALGORITHM,
+  SIMPLE_ALGORITHM,
+  REPLACEMENT_ALGORITHM,
+} from './fixtures/algorithms/algorithms';
+
+const menuGoTo = async (client, selector) => {
```

Comment:
Place it in `utils.js`, and you could add a new `menuGoToAlgorithms` with the selector already set.